db/changes.go

```diff
@@ -594,8 +760,14 @@ func (db *Database) SimpleMultiChangesFeed(chans base.Set, options ChangesOption
 				feeds, names = db.appendUserFeed(feeds, names, options)
 			}
 
-			// TODO: Use revocations option here in CBG-1367
-			_ = options.Revocations
+			if options.Revocations {
+				channelsToRevoke := db.user.RevokedChannels(options.Since.SafeSequence())
+				for channel, triggeredBy := range channelsToRevoke {
+					feed := db.buildRevokedFeed(db.changeCache.getChannelCache().getSingleChannelCache(channel), options, triggeredBy)
+					feeds = append(feeds, feed)
+					names = append(names, channel)
```

Comment:
Is names actually still used?  If it is, should we be modifying the name to indicate that this is a revocation feed, in the same way we're doing for 'late' sequence feeds?

Reply:
Doesn't look like it's used, or has been used for as far back as I can see in release history. Will remove it. Only reason I used it is I was following what others were doing.